src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -83,16 +142,53 @@ export const gravityStitchingEnvironment = (
           before: String
         ): UserAddressConnection
       }
+
+      extend type Partner {
+        searchCriteriaConnection(
+          first: Int,
+          last: Int,
+          before: String,
+          after: String,
+          artistID: ID,
+          highQuality: Boolean,
+          partnerID: String,
+          previewByArtist: Boolean
+          since: Int
+        ): SearchCriteriaConnection
```

Comment:
Added this here. This is how we connect the gravity field to the partner field (which lives in MP). Gotta use SDL and then wrangle it further down